git-rebase-via-merge.sh

```diff
@@ -168,15 +182,52 @@ fix_merge_conflicts(){
         echo
       fi
     elif [ "$input" = "a" ]; then
-        echo "Aborting merge."
-        git merge --abort
-        git checkout $current_branch
-        echo "Aborted."
-        exit 2
+      echo "Aborting merge."
+      git merge --abort
+      git checkout "$current_branch"
+      echo "Aborted."
+      exit 2
     else
-        echo "Invalid option."
-        echo "Type key 'c' - to Continue or 'a' - to Abort."
+      echo "Invalid option."
+      echo "Type key 'c' - to Continue or 'a' - to Abort."
+      echo
+    fi
+  done
+}
+
+fix_rebase_conflicts() {
```

Comment:
This function is almost an exact copy of `fix_merge_conflicts` but... for rebases. 😄